### DISCUSSION TITLE
使用百度网盘Webdav服务时报错

### DISCUSSION THREAD

**Discussion by wangty163** [AlistGo/alist#1257] [Invalid question]
【使用背景】
使用alist挂载了自己的百度网盘，同时打开了webdav服务，正常的上传、下载等操作均可以实现。
所以本人尝试使用群晖的Hyper Backup套件，将数据备份到百度网盘上（通过Alist的WebDav），但是发现会报错。我自己通过调用Alist的WebDav接口，发现是当上传一个空文件时候会报错，这个应该是百度网盘的上传接口的限制导致的。

【诉求】
能不能实现，过滤掉这些空文件，以防止百度网盘的接口报错呢？或者提供回调接口之类的，可以方便二次开发

**Comment by Yang-z** [comment]
应该是alist自动过滤空文件，导致群晖Hyper Backup报错。天翼云盘试了很多次都这样。希望alist不要有任何过滤，不然群晖会认为出错，一出错，全部要重来。。。
